transformer_sidecar/src/transformer_sidecar/object_store_manager.py

```diff
@@ -60,10 +64,12 @@ def upload_file(self, bucket, object_name, path):
                              "requestId": bucket, "object": result.object_name})
         except S3Error:
             self.logger.error("S3Error", exc_info=True)
```

Comment:
Does this swallow the message from the underlying `S3Error` or `MinioException`, possibly making it harder to understand what happened?

Reply:
Updated to log the stack trace and then include the original exception in the re-raised exception